core/controlplane/config/templates/cluster.yaml

```diff
@@ -836,7 +836,15 @@ worker:
 #     # See https://github.com/kubernetes-incubator/kube-aws/issues/230 for more information.
 #     rbac:
 #       enabled: true
-#
+#   # This flag disables automatic security group rule management for ELBs.  
+#   # It requires that the user has setup a rule that allows inbound traffic on kubelet ports 
+#   # from the local VPC subnet so that load balancers can access it.
+#   # Disable setting a new security group rule for every ELB.
+#   disableSecurityGroupIngress: false
```

Comment:
I want to keep the name the same as the command line flag passed to the controller.  Generally I'll use this pattern to prevent obfuscation for any power users who need to dig down to what a flag does.